Formula/javarepl.rb

```diff
@@ -6,9 +6,11 @@ class Javarepl < Formula
 
   bottle :unneeded
 
+  depends_on :java => "1.8"
+
   def install
     libexec.install "javarepl-#{version}.jar"
-    bin.write_jar_script libexec/"javarepl-#{version}.jar", "javarepl"
+    bin.write_jar_script libexec/"javarepl-#{version}.jar", "javarepl", :"java_version" => "1.8"
```

Comment:
Are the quotes around java_version needed?

Reply:
Nope. Amended.